libs/core/execution_base/tests/unit/CMakeLists.txt

```
@@ -15,6 +15,7 @@ set(tests
     execution_context
     get_env
     execute_may_block_caller
+    coroutine_traits
```

This should be added to the tests only if `HPX_WITH_CXX20_COROUTINES` is On. E.g. somethig like:
```suggestion
)
if(HPX_WITH_CXX20_COROUTINES)
  set(tests ${tests} coroutine_traits)
endif()
```

---

I have tried `HPX_HAVE_CXX20_COROUTINES` but it doesn't seem to work. All config being unchanged now suddenly it found no target `coroutine_traits_test`

---

Note, we generally use `HPX_WITH_XXX` during configuration time and a corresponding `HPX_HAVE_XXX` as preprocessor constants to be used during compilation.

---

https://github.com/STEllAR-GROUP/hpx/pull/5945/commits/c8090feefb12261d618a345e9d7302f03acf7584 fixed it for llvm/12.0.1 module